<!-- ultrasonic.launch -->
<launch>
  <!-- NÃ³ dos ultrasonicos -->
  <node pkg="robot_perception" type="i2c_ultrasonic.py" name="i2c_ultrasonic"/>
</launch>
